### PR TITLE
fix: add elb pool v3

### PR DESCRIPTION
**Pull request by houpeng80**
fixes https://github.com/FlexibleEngineCloud/terraform-provider-flexibleengine/issues/801

Test result:

make testacc TE
ST='./flexibleengine/acceptance' TESTARGS='-run TestAccElbV3Pool_basic'                                                      
==> Checking that code complies with gofmt requirements... 
TF_ACC=1 go test ./flexibleengine/acceptance -v -run TestAccElbV3Pool_basic -timeout 720m 
=== RUN   TestAccElbV3Pool_basic 
=== PAUSE TestAccElbV3Pool_basic 
=== CONT  TestAccElbV3Pool_basic 
--- PASS: TestAccElbV3Pool_basic (157.77s) 
PASS
ok      github.com/FlexibleEngineCloud/terraform-provider-flexibleengine/flexibleengine/acceptance      157.838s

### CODE REVIEW

File: docs/resources/lb_pool_v3.md [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+
+* `type` - (Required, String, ForceNew) The type of persistence mode. The current specification supports SOURCE_IP,
+  HTTP_COOKIE, and APP_COOKIE.
+
+* `cookie_name` - (Optional, String, ForceNew) The name of the cookie if persistence mode is set appropriately. Required
+  if `type = APP_COOKIE`.
+
+## Attributes Reference
+
+In addition to all arguments above, the following attributes are exported:
```
Comment (deer-hang) [inline docs/resources/lb_pool_v3.md:67]: the following attribute is exported

File: docs/resources/lb_pool_v3.md [hunk 2]
```diff
@@ -0,0 +1,85 @@
+---
+subcategory: "Dedicated Load Balance (Dedicated ELB)"
```
Comment (Jason-Zhang9309) [inline docs/resources/lb_pool_v3.md:2]: suggest to move other elb v3 resources to this subcategory,
that will make the docs structure clear

File: flexibleengine/acceptance/resource_flexibleengine_lb_pool_v3_test.go [hunk 3]
```diff
@@ -0,0 +1,119 @@
...
+  description     = "test description"
+  protocol        = "HTTP"
+  protocol_port   = 8080
+  loadbalancer_id = flexibleengine_lb_loadbalancer_v3.test.id
+
+  forward_eip = true
+
+  idle_timeout = 60
+  request_timeout = 60
+  response_timeout = 60
```
Comment (Jason-Zhang9309) [inline flexibleengine/acceptance/resource_flexibleengine_lb_pool_v3_test.go:90]: Please align the "="

File: docs/resources/lb_pool_v3.md (hunk 1 above)
Comment (houpeng80) [reply docs/resources/lb_pool_v3.md]: done

File: docs/resources/lb_pool_v3.md (hunk 2 above)
Comment (houpeng80) [reply docs/resources/lb_pool_v3.md]: done

File: flexibleengine/acceptance/resource_flexibleengine_lb_pool_v3_test.go (hunk 3 above)
Comment (houpeng80) [reply flexibleengine/acceptance/resource_flexibleengine_lb_pool_v3_test.go]: done

**Pull request closed without merging**